napari/utils/colormaps/vendored/cm.py

```diff
@@ -242,7 +241,6 @@ def to_rgba(self, x, alpha=None, bytes=False, norm=True):
 
         If norm is False, no normalization of the input data is
         performed, and it is assumed to be in the range (0-1).
-
```

Comment:
this is likely due to the fact that this docstring has no sections with headers otherwise it would leave a blankline.